<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PTZSettings</class>
 <widget class="QWidget" name="PTZSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PTZ Device Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="ptzConfigTab">
      <attribute name="title">
       <string>Configuration</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>P&amp;TZ Devices</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="deviceList">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="addPTZ">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Add PTZ Device</string>
               </property>
               <property name="accessibleName">
                <string>Add PTZ Device</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string notr="true">addIconSmall</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removePTZ">
               <property name="maximumSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="toolTip">
                <string>RemoveScripts</string>
               </property>
               <property name="accessibleName">
                <string>RemoveScripts</string>
               </property>
               <property name="text">
                <string notr="true"/>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
               <property name="themeID" stdset="0">
                <string notr="true">removeIconSmall</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="" native="true">
          <layout class="QVBoxLayout" name="propertiesLayout" stretch="0,0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QCheckBox" name="gamepadCheckBox">
             <property name="text">
              <string>Enable Gamepad</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="applyButton">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="close">
               <property name="text">
                <string>Close</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="aboutTab">
      <attribute name="title">
       <string>About</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="versionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
